Tests/SwiftCliCoreTests/InputValidatorTest.swift

```diff
@@ -4,27 +4,56 @@ import Nimble
 @testable import SwiftCliCore
 
 class ValidatorTests: XCTestCase {
+
+    private var inputValidator: InputValidator!
+
+    override func setUp() {
+        super.setUp()
+        inputValidator = InputValidator(patternNormalizer: PatternNormalizer(), nestedArrayBuilder: NestedArrayBuilder())
+    }
+
+    override func tearDown() {
+        inputValidator = nil
+        super.tearDown()
+    }
+
     func testInvalidStitchShouldThrowError() throws {
         let testPattern = ["g1 p1"]
         let err = InputError.invalidStitch(invalidStitch: "g1", rowLocation: 1)
-        expect { try InputValidator().inputValidation(pattern: testPattern, knitFlat: false) }.to(throwError(err))
+        expect { try self.inputValidator.inputValidation(
+            pattern: testPattern,
+            knitFlat: false)
+        }
+        .to(throwError(err))
     }
 
     func testEmptyRowShouldThrowError() throws {
         let testPattern = ["p1 p1", ""]
-        expect { try InputValidator().inputValidation(pattern: testPattern) }.to(throwError(InputError.emptyRow))
+        expect { try self.inputValidator.inputValidation(
+            pattern: testPattern)
+
+        }
```

Comment:
I know this is nit picky, but I think it serves a purpose for readability/future coders: if your closing bracket has it's own line, don't put anything on the same line as the opening bracket.  For example
```
expect { try self.inputValidator.inputValidation(
            pattern: testPattern)

        }
```
vs
```
expect { 
            try self.inputValidator.inputValidation(pattern: testPattern)
}
```

Reply:
Makes sense! Thanks for catching that.